### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by dyladan**
:robot: I have created a release *beep* *boop*
---

<details><summary>auto-instrumentations-node: 0.46.1</summary>

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * @opentelemetry/instrumentation-aws-lambda bumped from ^0.41.0 to ^0.41.1
    * @opentelemetry/instrumentation-connect bumped from ^0.36.0 to ^0.36.1
    * @opentelemetry/instrumentation-express bumped from ^0.38.0 to ^0.39.0
    * @opentelemetry/resource-detector-aws bumped from ^1.4.2 to ^1.5.0
</details>

<details><summary>baggage-span-processor: 0.2.0</summary>

### Features

* Add new baggage span processor package ([#2114](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2114)) ([695177b](https://github.com/open-telemetry/opentelemetry-js-contrib/commit/695177b69cfb51f70064dc1ee615383d560aa62a))
</details>

<details><summary>instrumentation-aws-lambda: 0.41.1</summary>

### Bug Fixes

* **aws-lambda:** update semconv package and readme ([#2181](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2181)) ([b432929](https://github.com/open-telemetry/opentelemetry-js-contrib/commit/b432929f4b5682e7879c50682547cea25790c588))
</details>

<details><summary>instrumentation-connect: 0.36.1</summary>

### Bug Fixes

* **instrumentation-connect:** add source maps to package ([#2200](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2200)) ([1ffc631](https://github.com/open-telemetry/opentelemetry-js-contrib/commit/1ffc631a7fed330b1677ed17bad87b305edcbed7))
</details>

<details><summary>instrumentation-express: 0.39.0</summary>

### Features

* **instrumentation-express:** Support non-string routes ([#2008](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2008)) ([525bbba](https://github.com/open-telemetry/opentelemetry-js-contrib/commit/525bbba67accc20eeb96899ab7dc9ccb125a4b2d))
</details>

<details><summary>resource-detector-aws: 1.5.0</summary>

### Features

* add support for cloud.resource_id to AWS ECS detector ([#1936](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1936)) ([cc71492](https://github.com/open-telemetry/opentelemetry-js-contrib/commit/cc714922b931ce39595ef0b257114b4e797ba6a9))
</details>

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Review by pichlermarc — APPROVED** [review]

File: detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md [hunk 1]
```diff
@@ -66,6 +66,13 @@
   * devDependencies
     * @opentelemetry/contrib-test-utils bumped from ^0.35.1 to ^0.36.0
```
Comment (trentm) [inline detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md:68]: What's going on with all the `### Dependencies` sections above here?

I see it in auto-instrumentations-node/CHANGELOG.md below as well.

It's in these files in this repo:

```
% git ls-files | rg CHANGELOG.md | while read f; do if [[ $(cat $f | head -3 | tail -1) == "### Dependencies" ]]; then echo $f; fi ; done
detectors/node/opentelemetry-resource-detector-alibaba-cloud/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-azure/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-container/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-gcp/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-instana/CHANGELOG.md
metapackages/auto-instrumentations-node/CHANGELOG.md
metapackages/auto-instrumentations-web/CHANGELOG.md
packages/opentelemetry-propagation-utils/CHANGELOG.md
plugins/node/instrumentation-amqplib/CHANGELOG.md
plugins/node/instrumentation-lru-memoizer/CHANGELOG.md
plugins/node/instrumentation-mongoose/CHANGELOG.md
plugins/node/instrumentation-socket.io/CHANGELOG.md
plugins/node/instrumentation-tedious/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-cassandra/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-memcached/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-mongodb/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-mysql/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-mysql2/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-pg/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-redis-4/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-redis/CHANGELOG.md
```

Going back looking at changes to `detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md` to see when a `### Dependencies` section was incorrectly added to the top (not under an h2), the last good PR was:

    chore: release main (#1590)

then starting with this release PR it started incorrectly adding that section:

    chore: release main (#1683)

and continued for a number of releases:

    ...
    chore: release main (#1890)

until fixed by the time this release PR was made:

    chore: release main (#1935)


Anyway... this is independent of this PR. I'll open a separate issue.

**Review by trentm — APPROVED** [review]

File: detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md (hunk 1 above)
Comment (trentm) [reply detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md:68]: https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2207 opened for this.

**Comment by dyladan** [comment]
:robot: Release is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/resource-detector-aws-v1.5.0 :sunflower:

**Comment by dyladan** [comment]
:robot: Release is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/auto-instrumentations-node-v0.46.1 :sunflower:

**Comment by dyladan** [comment]
:robot: Release is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/baggage-span-processor-v0.2.0 :sunflower:

**Comment by dyladan** [comment]
:robot: Release is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/instrumentation-aws-lambda-v0.41.1 :sunflower:

**Comment by dyladan** [comment]
:robot: Release is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/instrumentation-connect-v0.36.1 :sunflower:

**Comment by dyladan** [comment]
:robot: Release is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/instrumentation-express-v0.39.0 :sunflower:

**Pull request merged**